boolean logic added

coltonious opened the pull request:
added boolean logic to the requires lines so that if/when the manual devs phase out old requires lines, this apworld can be prepared.

added The Animatronic, which releases on June 12th I believe

coltonious committed:
added boolean logic to the requires lines, so that if they phase out old logic this apworld can be prepared.

coltonious committed:
Add "The Animatronic", willard appleton from fred five bear five freddy nights

FuzzyGamesOn commented on data/locations.json:
```diff
@@ -6,7 +6,7 @@
         ],
         "requires": [
-            "A Nurse's Calling"
+            "|A Nurse's Calling|"
```
Randomly took a peek at this while looking for something else and figured I'd comment. :smiley: When you use boolean logic, you need to remove the square brackets (`[]`) around the requires as well.

All of the locations changed or added in this PR need this adjusted as well.

coltonious commented:
ope, you're just right. Ill fix that right now and redo the PR lmao

On Mon, Jun 16, 2025 at 4:00 PM FuzzyGames ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In data/locations.json
> <https://github.com/nivomi/deadbydaylight_manual_archipelago/pull/1#discussion_r2150877476>
> :
>
> > @@ -6,7 +6,7 @@
>              "Nurse, The"
>          ],
>          "requires": [
> -            "A Nurse's Calling"
> +            "|A Nurse's Calling|"
>
> Randomly took a peek at this while looking for something else and figured
> I'd comment. 😃 When you use boolean logic, you need to remove the square
> brackets ([]) around the requires as well.
>
> All of the locations changed or added in this PR need this adjusted as
> well.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nivomi/deadbydaylight_manual_archipelago/pull/1#pullrequestreview-2933503878>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5FL45AG4PDLBNAWWYKZA5D3D4V7TAVCNFSM6AAAAAB6KFH4A6VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDSMZTGUYDGOBXHA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>

coltonious committed:
fixed the requires lines that i had messed up. sorry about that!

coltonious committed:
also forgot that the way you name the "game" needed to be changed! whoops!

coltonious commented:
uhhh i just realized that i somehow cloned a repo that didn't have the items.json? so i guess ill add that back in and add the fnaf ones

coltonious committed:
no clue why this wasn't in the cloned repository. fixed and added in the animatronic

coltonious committed:
added in the animatronic stuff

coltonious committed:
somehow enduring's requires got scuffed

coltonious commented:
okay it should finally be good. sorry about all that, ladies and germs

coltonious committed:
copied the new manual client information over to allow for playing on archi 6.2

coltonious committed:
vommy mommy 2.0 added

coltonious committed:
fixed the furtive chase misspelling

coltonious committed:
forgot to change it to c37

coltonious committed:
i don't know why UWUVCI was added to this repo? that was strange

coltonious committed:
replaced a "place_item" with a "requires". That's what I get for letting github autofill do work for me.

coltonious committed:
The Henry has turned back the clock!

coltonious committed:
forgot to change game name for c38

coltonious committed:
forgot to add the category for the first

coltonious committed:
made a general filler item, and moved -1 hook to items.json at the request of Wabadjakk

hopefully correctly added regions at the request of MarioHacker

Fixed furtive chase to not have the weird lowercase f in "furtive"

coltonious committed:
hopefully fixed the error caused by a lowercase f that I must've missed in that last update

cdhowie committed:
Add readme

coltonious committed:
Merge pull request #1 from cdhowie/master

Added readme to clarify some issues and explain how the manual is intended to work

Thank you cdhowie :)

coltonious committed:
Fixed only 1 "-1 hook for victory" genning, as well as changed the names of removed license perks to show both own and non-own names. Hopefully that doesn't break anything.

coltonious committed:
Merge branch 'master' of https://github.com/coltonious/deadbydaylight_manual_archipelago-update_helper_fork-

coltonious committed:
I guess archipelago doesn't like colons in item names. fixed.

coltonious committed:
Major changes: added the guy with the machete and his perks

Minor changes: updated the max killer choice to allow for ALL killers, something I've been forgetting to do.

coltonious committed:
I forgor to change the chapter number

coltonious committed:
changed the name for hexes slightly in locations since i forgot to do it earlier, leading to generation error

coltonious committed:
completely forgor to do locations, also forgot about capitalization on some hexes
